yt_dlp/extractor/eplus.py

```diff
@@ -29,14 +36,99 @@ class EplusIbIE(InfoExtractor):
             'No video formats found!',
             'Requested format is not available',
         ],
+    }, {
+        'url': 'https://live.eplus.jp/sample',
+        'info_dict': {
+            'id': 'stream1ng20210719-test-005',
+            'title': 'Online streaming test for DRM',
+            'live_status': 'was_live',
+            'release_date': '20210719',
+            'release_timestamp': 1626703200,
+            'description': None,
+        },
+        'params': {
+            'skip_download': True,
+            'ignore_no_formats_error': True,
+        },
+        'expected_warnings': [
+            'Could not find the playlist URL. This event may not be accessible',
+            'No video formats found!',
+            'Requested format is not available',
+            'This video is DRM protected',
+        ],
+    }, {
+        'url': 'https://live.eplus.jp/2053935',
+        'info_dict': {
+            'id': '331320-0001-001',
+            'title': '丘みどり2020配信LIVE Vol.2 ～秋麗～ 【Streaming+(配信チケット)】',
+            'live_status': 'was_live',
+            'release_date': '20200920',
+            'release_timestamp': 1600596000,
+        },
+        'params': {
+            'skip_download': True,
+            'ignore_no_formats_error': True,
+        },
+        'expected_warnings': [
+            'Could not find the playlist URL. This event may not be accessible',
+            'No video formats found!',
+            'Requested format is not available',
+        ],
     }]
 
+    _USER_AGENT = 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/119.0.0.0 Safari/537.36 Edg/119.0.0.0'
+    _TEST_EVENT_URL = 'https://live.eplus.jp/2053935'
+
+    def _perform_login(self, id, password):
+        webpage, urlh = self._download_webpage_handle(
+            self._TEST_EVENT_URL, None, note='Getting auth status', errnote='Unable to get auth status')
+        if urlh.url.startswith(self._TEST_EVENT_URL):
+            # already logged in
+            return
+
+        cltft_token = self._hidden_inputs(webpage).get('Token.Default')
+        if not cltft_token:
+            raise ExtractorError('Unable to get X-CLTFT-Token', expected=False)
+        self._set_cookie('live.eplus.jp', 'X-CLTFT-Token', f'Token.Default={cltft_token}')
+
+        login_json = self._download_json(
+            'https://live.eplus.jp/member/api/v1/FTAuth/idpw', None,
+            note='Sending pre-login info', errnote='Unable to send pre-login info', headers={
+                'Content-Type': 'application/json; charset=UTF-8',
+                'Referer': urlh.url,
+                'X-Cltft-Token': f'Token.Default={cltft_token}',
+                'Accept': '*/*',
+            }, data=json.dumps({
+                'loginId': id,
+                'loginPassword': password,
+            }).encode())
+        if not login_json.get('isSuccess', None):
```

Comment:
Thanks. I should have read the docs before adding the default value.

> [`get(key[, default])`](https://docs.python.org/3/library/stdtypes.html#dict.get)
> 
> Return the value for key if key is in the dictionary, else default. If default is not given, it defaults to `None`, so that this method never raises a [KeyError](https://docs.python.org/3/library/exceptions.html#KeyError).